tests/ensure_provider_tests.py

```diff
@@ -72,11 +72,17 @@
     "playbooks/tests_team.yml": {
         EXTRA_RUN_CONDITION: "ansible_distribution != 'Fedora'",
     },
+    "playbooks/tests_team_exists.yml": {
+        EXTRA_RUN_CONDITION: "ansible_distribution != 'Fedora'",
```

Comment:
Why is this needed?

Reply:
tested, it is not needed, updated the commit already